### DISCUSSION TITLE
What is the best way to organize vim-plugins in NvChad fashion ?

### DISCUSSION THREAD

**Discussion by benzwt** [NvChad/NvChad#2146] [Q&A] [answered]
I'm going to install some vim-plugins which are written in vim-script.
I'm highly prefer the layout of NvChad: putting custom stuff in custom/.

My vim-plugins need some configurations, such as
``` vim
let g:var="......."
```
1. Where should I put those configuration files ?
2. can I use replace the `let syntax` with lua lang ?

thanks!

**Comment by siduck** [comment]
@benzwt first of all which vim plugin are you trying to install?

**Reply by benzwt** [reply]
Here is my list
1. Vim-easy-align
2. Vim-Gutentags
3. Gutentags-plus

**Reply by siduck** [reply, marked as answer]
ohh cool, i thought you wanted to install some syntax highlighting plugins lol. 

just put all your globals in the init function of your plugin, like

```lua
{  
   "vim-easy-align", 
   init = function() 
 --   let g:easy_align_interactive_modes = ['l', 'r']  
    vim.g.easy_align_interactive_modes = {'l', 'r'} 
    end,
    event =   "VeryLazy"
}
